spec/controllers/queue_items_controller_spec.rb

```diff
@@ -102,6 +102,102 @@
       delete :destroy, id: 1
       expect(response).to redirect_to sign_in_path      
     end
+
+    it 'normalize the remaining queue items' do
+      user = Fabricate(:user)
+      session[:user_id] = user.id
+      queue_item = Fabricate(:queue_item, user: user, video: Fabricate(:video))
+      queue_item2 = Fabricate(:queue_item, user: user, video: Fabricate(:video))
+      delete :destroy, id: queue_item.id
+      expect(QueueItem.first.position).to eq(queue_item2.reload.position)
```

Comment:
And....now I just realized that those specs are for a post and this is for a delete. I understand now why you did what you did. Please ignore all three of these comments lol.